velox/type/DecimalUtil.cpp

```diff
@@ -54,41 +54,42 @@ const int128_t DecimalUtil::kPowersOfTen[]{
     1,
     10,
     100,
-    1000,
-    10000,
-    100000,
-    1000000,
-    10000000,
-    100000000,
-    1000000000,
-    10000000000,
-    100000000000,
-    1000000000000,
-    10000000000000,
-    100000000000000,
-    1000000000000000,
-    10000000000000000,
-    100000000000000000,
-    1000000000000000000,
-    1000000000000000000 * (int128_t)10,
-    1000000000000000000 * (int128_t)100,
-    1000000000000000000 * (int128_t)1000,
-    1000000000000000000 * (int128_t)10000,
-    1000000000000000000 * (int128_t)100000,
-    1000000000000000000 * (int128_t)1000000,
-    1000000000000000000 * (int128_t)10000000,
-    1000000000000000000 * (int128_t)100000000,
-    1000000000000000000 * (int128_t)1000000000,
-    1000000000000000000 * (int128_t)10000000000,
-    1000000000000000000 * (int128_t)100000000000,
-    1000000000000000000 * (int128_t)1000000000000,
-    1000000000000000000 * (int128_t)10000000000000,
-    1000000000000000000 * (int128_t)100000000000000,
-    1000000000000000000 * (int128_t)1000000000000000,
-    1000000000000000000 * (int128_t)10000000000000000,
-    1000000000000000000 * (int128_t)100000000000000000,
-    1000000000000000000 * (int128_t)1000000000000000000,
-    1000000000000000000 * (int128_t)1000000000000000000 * (int128_t)10};
+    1'000,
```

Comment:
I made this change to enhance the readability. But this is not required for this PR.

Reply:
@majetideepak Thank you for clarifying. Next time, consider making such changes in separate PRs to make reviewing easier.

Reply:
@mbasmanova will do. I initially touched on this part of the code and made this change. Later this change was not required :), but decided to leave it since the diff was small.